cfgen/cfgen

```diff
@@ -2420,21 +2426,26 @@ def build_cluster(cluster_desc: Dict[str, Any]) -> Cluster:
         node_id = ConfNode.build(conf, root_id, node)
         encl_id = ConfEnclosure.build(conf, rack_id, node_id)
         ctrl_ids: List[Tuple[Optional[Oid], Any]] = []
-        for m0d in node.get('m0_servers', []):
-            if m0d['_io_disks']:
-                ctrl_ids.append((ConfController.build(conf, encl_id), m0d))
-            else:
-                ctrl_ids.append((None, m0d))
+        m0srv = node['m0_servers']
```

Comment:
..and here also

Reply:
Done!
As discussed with Mandar, this possibility need to be verified and and tested and improvement done using other JIRA.